backend/Cargo.toml

```diff
@@ -46,8 +46,10 @@ dotenvy = "0.15.0"
 typeshare = "1.0.0"
 derive_more = "0.99.17"
 uuid = { version = "1.3.2", features = ["serde", "v4"] }
+postgis_diesel = "2.1.0"
 log = "0.4.17"
 env_logger = "0.10.0"
+postgis_diesel = "2.1.0"
```

Comment:
Added twice. Remove one of them. Also move it up to the `# Data` section together with the other diesel dependencies.

Reply:
I don't even know how that happened. I guess it slipped through during the merge.